root/statistics/Edits.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l as lMbServer} from '../static/scripts/common/i18n';
+import {l_statistics as l} from '../static/scripts/common/i18n/statistics';
+import {withCatalystContext} from '../context';
+import manifest from '../static/manifest';
+
+import {formatCount, formatPercentage} from './utilities';
+import StatisticsLayout from './StatisticsLayout';
+
+type EditCategoryT = {|
+  +edit_name: string,
+  +edit_type: string,
+|};
+
+type EditsStatsT = {|
+  +$c: CatalystContextT,
+  +dateCollected: string,
+  +stats: {[string]: number},
+  +statsByCategory: {[string]: $ReadOnlyArray<EditCategoryT>},
+|};
+
+const Edits = ({$c, dateCollected, stats, statsByCategory}: EditsStatsT) => {
+  console.log(stats);
+  return (
+    <StatisticsLayout fullWidth page="edits" title={l('Edits')}>
+      {manifest.css('statistics')}
```

Comment:
`StatisticsLayout` already includes the CSS, so you can remove this plus the `manifest` import at the top (I'll have to do it in my Webpack PR in any case)

Reply:
Did that for every file in a second commit. One of the files (root/statistics/LanguagesScripts.js) also includes {manifest.js('statistics')}, which is not in the Layout, so I left that in.